app/models/formal_group.rb

```diff
@@ -48,7 +48,7 @@ class FormalGroup < Group
   belongs_to :default_group_cover
 
   has_many :subgroups,
-           -> { where(archived_at: nil).order(:name) },
```

Comment:
I agree with this, but wonder if we need to check the `/g/:key/subgroups` query to make sure it's still returning the same thing?